d3-scripts/src/d3_scripts/resolve_behaviour_rules.py

```diff
@@ -0,0 +1,33 @@
+import networkx as nx
+from .check_behaviours_resolve import BehaviourMap
+from iteration_utilities import unique_everseen
+from typing import List, Dict
+
+
+def resolve_behaviour_rules(claim: Dict, claim_map: BehaviourMap, claim_graph: nx.DiGraph) -> List[Dict]:
+    """
+    Resolve rules which apply for behaviour claim, and checks that any named rules exist and that
+    there are no duplicate rule names in claim behaviour.
+
+    Args:
+        claim: The D3 behaviour claim to resolve behaviour for
+        claim_tree: The claim inheritance tree for the claim
```

Comment:
Out-of-date docstring
```suggestion
        claim_map: Map of D3 claim GUID to D3 behaviour claim json
        claim_graph: Claim inheritance graph that shows this claim's parents
```